packages/ai/src/steps/generateCodeStep.ts

```diff
@@ -17,7 +17,76 @@ export const generateNewSquiggleCodePrompt = (prompt: string): PromptPair => {
 7. Add at least one test per complex function, using sTest, to test functionality, if the prompt requests more than 10 lines of code (explicitly or implicitly). Assign sTest.describe() blocks to variables, like this: \`function1_tests = sTest.describe(...)\`. Note that you cannot use multiple sTest.expect() calls in a single test() block.
 8. If the prompt requests changes to existing code, try to keep somewhat close to that code.
 9. Use \`@name\` annotations for concise descriptions; use \`@doc\` when further details are necessary.
-10. Write a comment at the top of the code that summarizes the code and its purpose.
+
+**Dicts vs. Blocks**
+## Dictionaries and Blocks
```

Comment:
I manually copied and pasted this from the main LLM prompt. It seemed important to reiterate it here so that it receives more attention. But I wasn't sure how to best store it in @quri/content in a way that could easily be reused in both places. Can revisit later.